service/src/test/java/bio/terra/cbas/util/methods/TestBackfillOriginalWorkspaceIdService.java

```diff
@@ -0,0 +1,298 @@
+package bio.terra.cbas.util.methods;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.cbas.config.CbasContextConfiguration;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunSetDao;
+import bio.terra.cbas.models.CbasRunSetStatus;
+import bio.terra.cbas.models.Method;
+import bio.terra.cbas.models.MethodVersion;
+import bio.terra.cbas.models.RunSet;
+import bio.terra.cbas.util.BackfillOriginalWorkspaceIdService;
+import java.time.OffsetDateTime;
+import java.util.Arrays;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+public class TestBackfillOriginalWorkspaceIdService {
+  @MockBean RunSetDao runSetDao;
+  @MockBean MethodDao methodDao;
+  @MockBean MethodVersionDao methodVersionDao;
+  @MockBean CbasContextConfiguration cbasContextConfig;
+
+  @Test
+  void testBackfillMethods() {
+    CbasContextConfiguration cbasContextConfig = mock(CbasContextConfiguration.class);
```

Comment:
Because you're making these mocks explicitly and then [calling](https://github.com/DataBiosphere/cbas/pull/255/files#diff-7ed4393471905195d5e9217a76c55a3421e7141b80b72b61c22cb4f753da4535R49) the standard constructor, I don't think they need to be marked as mockbeans above. That would be necessary if you were letting spring autowire them through